test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -1346,18 +1339,13 @@ TEST_F(PickFirstTest, PendingUpdateAndSelectedSubchannelFails) {
   gpr_log(GPR_INFO, "Phase 3: Stopping first server.");
   servers_[0]->Shutdown();
   WaitForChannelNotReady(channel.get());
-  // TODO(roth): This should always return CONNECTING, but it's flaky
-  // between that and TRANSIENT_FAILURE.  I suspect that this problem
-  // will go away once we move the backoff code out of the subchannel
-  // and into the LB policies.
   EXPECT_THAT(channel->GetState(false),
-              ::testing::AnyOf(GRPC_CHANNEL_CONNECTING,
-                               GRPC_CHANNEL_TRANSIENT_FAILURE));
-  // Now start the second server.
-  gpr_log(GPR_INFO, "Phase 4: Starting second server.");
-  StartServer(1);
-  // The channel should go to READY state and RPCs should go to the
-  // second server.
+              ::testing::AnyOf(GRPC_CHANNEL_CONNECTING));
```

Comment:
Don't need `AnyOf()` here anymore.  Instead, you can just say `EXPECT_EQ(channel->GetState(false), GRPC_CHANNEL_CONNECTING)`.

Reply:
done.